File: Group_One_Server/src/main/java/SharedDataTypes/TicketObject.java
package SharedDataTypes;

import SharedDataTypes.SensorObjects.SensorObject;
import SharedDataTypes.SensorObjects.VirtualSensorObject;

public class TicketObject {

    public int ticketID;
    public String fieldNotes;
    public SensorObject sensorInfo;
    public String creationDate;
    public TicketState ticketState;
    public String assignedUserID;
    public int assignedTimeSlot;
    public boolean isSent = false;


}

File: g1-gui-app/Assets/Scripts/DataTypes/TicketObject.cs
using System;
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

[Serializable]
public class TicketObject
{
    public int ticketID;
    public string fieldNotes;
    public VirtualSensorData sensorInfo;
    public string creationDate;
    public string ticketState;
    public string assignedUserID;
    public int assignedTimeSlot;

}
